praw/const.py

```diff
@@ -1,7 +1,7 @@
 """PRAW constants."""
 from .endpoints import API_PATH  # noqa: F401
 
-__version__ = "6.5.2.dev0"
+__version__ = "6.6.0.dev0"
```

Comment:
Why are we version bumping?

Reply:
We're adding a new feature, I thought that a minor version bump was in order since it's backwards compatible (otherwise it would be a major bump) but this is the development branch so if you want we can merge all changes on the next release and update the version there.

Reply:
Multiple features get combined into a release. Each feature does not warrant a new release.

Reply:
This isn't a real version bump so this is fine. It's indicating that this is the first change on the master branch that will result in a minor version. I think that's appropriate.

Reply:
All right reverted the revert, rebumped the debumped bumpity doo, we're good to go!